example-playbooks/replace_node/replace_node.yml

```diff
@@ -30,26 +30,38 @@
           become: true
 
         - name: Run nodetool drain
-          async_task:
-            shell: |
-              nodetool drain
-            alias: drain_scylla
-            async: 60
-            retries: 60
-            delay: 1
-          ignore_errors: true
-          register: _drain_scylla_output
+          block:
+            - name: Async nodetool drain
+              async_task:
+                shell: |
+                  nodetool drain
+                alias: drain_scylla
+                async: 60
+                retries: 60
+                delay: 1
+              ignore_errors: true
+              register: _drain_scylla_output
+
+            - name: Cleanup job alias
+              shell: |
+                rm -rf "{{ _drain_scylla_output.alias_path }}"
```

Comment:
Don't you want to do that only if the task fails? I'm sure there is a field in `_drain_scylla_output` that has the exit status, no?

Reply:
Fixed